src/main/java/mate/academy/hibernate/relations/dao/impl/ActorDaoImpl.java

```diff
@@ -12,11 +15,32 @@ public ActorDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Actor add(Actor actor) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = factory.openSession();
```

Comment:
Consider using `getSessionFactory()` instead of directly accessing `factory` to maintain consistency and clarity, especially if `factory` is a protected field in the superclass `AbstractDao`.